packages/eslint-config-small-ads/tsconfig.json

```diff
@@ -0,0 +1,41 @@
+{
```

Comment:
As I see it this while file can be deleted. We will be having a tsconfig for each package.

Reply:
Hmm, it is true. I deleted.